packages/console/src/pages/SignInExperience/components/Preview/index.tsx

```diff
@@ -120,9 +120,9 @@ const Preview = ({ signInExperience, className }: Props) => {
 
     previewRef.current?.contentWindow?.postMessage(
       { sender: 'ac_preview', config },
-      window.location.origin
+      userEndpoint?.origin ?? ''
```

Comment:
nit: simply userEndpoint?

Reply:
could be `undefined` and doesn't fit the signature